### DISCUSSION TITLE
Material durante el curso

### DISCUSSION THREAD

**Discussion by Alejandro-Ari** [Launch-X-Latam/MisionFrontEnd#65] [Q&A]
Buenas tardes. ¿Creen que será posible llevar este curso con una Ipad 9th generación?, es decir que se pueda trabajar con la posibles applicaciones, compiladores, etc...

**Comment by DthArgelius** [comment]
para la parte front end creo que si podrias pero para el back end ya no.. pero creo mejor que para mayor comodidad debes conseguirte una laptop o pc de escritorio

**Reply by Alejandro-Ari** [reply]
ok vale, lo considerare, gracias!

**Comment by JosafatJimenezB** [comment]
Preferentemente te recomiendo utilizar una pc mas que nada para una mayor comodidad al realizar las practicas, donde la vas a requerir mas es en la parte de los frameworks ya que hay que tener dependencias como **node** y las de **vue o react** y estas me parece que solo son para pc

**Reply by Alejandro-Ari** [reply]
Lo considerare, gracias 👍

**Comment by maxjhael** [comment]
No he tenido una Ipad, pero si te permite abrir la terminal y ponerle vscode o el editor que tu gustes y aparte le compras un teclado que le puedas adaptar estoy seguro que podras usarla para ese fin
Y si desearas adquirir una pc o laptop para formarte, te recomendaria algo con 8 gigas de ram y algo con 2 ghz de procesamiento, con eso puedes tener un aprendizaje fluido
